lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -81,12 +95,29 @@ def active_step_by_step
         @active_step_navs.first
       end
 
+      def secondary_step_by_step?
+        secondary_step_by_steps.any? && step_navs_combined_list.count === 0
```

Comment:
Ah, I think `step_navs_combined_list.count === 0` is being used to check whether there are any primary step by steps? 

Perhaps this check could be moved to a method, so it's easier to see what it does, e.g. 
```
def primary_step_navs?
  step_navs_combined_list.count > 0
end
```